hw/dv/data/common_sim_cfg.hjson

```diff
@@ -137,6 +137,6 @@
   vcs_cov_excl_files: ["{tool_srcs_dir}/common_cov_excl.el"]
 
   // Project defaults for Xcelium
-  // xcelium_cov_cfg_file: "-covfile {tool_srcs_dir}/cover.ccf"
-  // xcelium_cov_refine_file: ["{tool_srcs_dir}/common_cov.vRefine"]
+  xcelium_cov_cfg_file: "-covfile {tool_srcs_dir}/cover.ccf"
```

Comment:
These dont exist at the moment, right?

Reply:
oops, sorry I was experimenting around.. Only mean to turn on the xcelium coverage. thanks for the catch.